File: src/test/java/com/github/ccguyka/showupdates/DependencyBuilder.java
package com.github.ccguyka.showupdates;

import org.apache.maven.artifact.Artifact;
import org.apache.maven.model.Dependency;

class DependencyBuilder {

    public static Dependency aDependency(Artifact artifact) {
        Dependency dependency = new Dependency();
        dependency.setGroupId(artifact.getGroupId());
        dependency.setArtifactId(artifact.getArtifactId());
        dependency.setVersion(artifact.getVersion());
        dependency.setScope(artifact.getScope());
        dependency.setType(artifact.getType());
        dependency.setClassifier(artifact.getClassifier());
        return dependency;
    }
}

File: src/test/kotlin/com/github/ccguyka/showupdates/DependencyBuilder.kt
package com.github.ccguyka.showupdates

import org.apache.maven.artifact.Artifact
import org.apache.maven.model.Dependency

internal object DependencyBuilder {

    @JvmStatic
    fun aDependency(artifact: Artifact): Dependency {
        val dependency = Dependency()
        dependency.groupId = artifact.groupId
        dependency.artifactId = artifact.artifactId
        dependency.version = artifact.version
        dependency.scope = artifact.scope
        dependency.type = artifact.type
        dependency.classifier = artifact.classifier
        return dependency
    }
}
